Makefile

```
@@ -162,32 +171,37 @@ endif
 
 .PHONY: test
 test:
-	gotestsum --format pkgname -- ./... $(TEST_FLAGS)
+	gotestsum --format pkgname -- $(DEFAULT_TEST_DIRECTORIES) $(TEST_FLAGS)
 
 # Only build the tests (don't execute them).
 .PHONY: test\:build
 test\:build:
-	gotestsum --format pkgname -- ./... $(TEST_FLAGS) -run=nope
+	gotestsum --format pkgname -- $(DEFAULT_TEST_DIRECTORIES) $(TEST_FLAGS) -run=nope
 
 .PHONY: test\:ci
 test\:ci:
-	DEFRA_BADGER_MEMORY=true DEFRA_BADGER_FILE=true $(MAKE) test:names
+	DEFRA_BADGER_MEMORY=true DEFRA_BADGER_FILE=true $(MAKE) test:all
 
 .PHONY: test\:go
 test\:go:
-	go test ./... $(TEST_FLAGS)
+	go test $(DEFAULT_TEST_DIRECTORIES) $(TEST_FLAGS)
 
 .PHONY: test\:names
 test\:names:
-	gotestsum --format testname -- ./... $(TEST_FLAGS)
+	gotestsum --format testname -- $(DEFAULT_TEST_DIRECTORIES) $(TEST_FLAGS)
+
+.PHONY: test\:all
+test\:all:
+	@$(MAKE) test:names
+	@$(MAKE) test:lens
```

todo: I think only if the first line is using another make target name (`$(make)`) then we need this (only in first line).

```suggestion
	$(MAKE) test:lens
```

---

I don't think it matters.  If I remove the `@` it essentially couples the line to the one above it, as if the line above (with `@`) is removed, then this line must be changed (to add an `@`).  Leaving as-is